### PR TITLE
Fix issue with expected param in script

### PR DESCRIPTION
**Pull request by okozlovlux**
Add request type LOCK_SCREEN_ICON_URL to default policies section
in json file

### CODE REVIEW

**Comment by okozlovlux** [comment]
@dboltovskyi please review

File: files/ptu.json [hunk 1]
```diff
@@ -2299,7 +2299,8 @@
         "RequestType":[
         	"PROPRIETARY",
         	"HTTP",
-        	"QUERY_APPS"
+        	"QUERY_APPS",
+          "LOCK_SCREEN_ICON_URL"
```
Comment (dboltovskyi) [inline files/ptu.json]: @okozlovlux Please use the same indent as for other elements

File: test_scripts/Polices/build_options/022_ATF_P_TC_PTU_Merge_Into_Local_PT_HTTP.lua [hunk 2]
```diff
@@ -227,4 +219,4 @@ function Test.Postcondition_Stop_SDL()
   StopSDL()
 end
 
-return Test
+return Test
```
Comment (dboltovskyi) [inline test_scripts/Polices/build_options/022_ATF_P_TC_PTU_Merge_Into_Local_PT_HTTP.lua]: @okozlovlux Please add new line at the end of the script

File: files/ptu.json (hunk 1 above)
Comment (okozlovlux) [inline files/ptu.json]: @dboltovskyi , fixed 9df3d9f

File: test_scripts/Polices/build_options/022_ATF_P_TC_PTU_Merge_Into_Local_PT_HTTP.lua (hunk 2 above)
Comment (okozlovlux) [inline test_scripts/Polices/build_options/022_ATF_P_TC_PTU_Merge_Into_Local_PT_HTTP.lua]: @dboltovskyi , fixed 9df3d9f

**Pull request merged**